src/PerformanceTests/Tests/Enabled/CSharp_CA1416.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
```

Comment:
I am wondering if it would be worth to have a specific `.editorconfig` in the `PerformanceTests` section.

Reply:
➡️ These will become unnecessary after Roslyn.Diagnostics.Analyzers is updated and the new RelaxNamingSuppressor rules take effect.